install/Knative-with-IKS.md

```diff
@@ -135,8 +135,14 @@ components, the recommended configuration for a cluster is:
 
 Knative depends on Istio.
 
-1.  Install Istio:
+1.  Install Istio.
+    Install the Istio CRDs first so that Kubernetes has time to create
+        the resource definitions. Sometimes the deployment of `istio.yaml`
+	(which includes those CRD definitions) fails because the types are
+	have not finished being defined by the CRD controller.
```

Comment:
If the istio.yaml deployment fails, what should you do? Wait and try again? If so, can we include that instruction?